deployable/module-template/README.md

```diff
@@ -0,0 +1,7 @@
+<!-- Update the title -->
+# Deployable Module Template
+
+<!-- Update the below text with the name of the module  -->
+
+A thin wrapper around the [terraform-ibm-module-template](../../) module which includes a provider configuration meaning it can be deployed as is.
```

Comment:
```suggestion
A wrapper for the [terraform-ibm-module-template](../../) module so that it can be deployed.
```

`module-template`: we say that they need to rename this directory ??

I'm not sure whether we expect this readme to show up in the catalog. If it does, we might want catalog-specific info for the consumer, like what you can do with it. We also might want template text about how this is a basic example or whatever.

- `thin`: doesn't add anything substantial does it?
- I'm not sure that saying that it includes the provider is helpful to consumers. Developers will get it already, right?
- `...can be deployed.` 
    I debated whether we need to say where (in the IBM Cloud catalog).

Reply:
I don't think this would / should show in catalog. If we are going with the direction of having it as part of the module itself, then only the module readme would show, not the deployable module readme